src/etc/inc/notices.inc

```diff
@@ -410,10 +410,60 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 	log_error(sprintf(gettext("Message sent to %s OK"), $to));
 	return;
 }
+/****f* notices/notify_via_telegram
+ * NAME
+ *   notify_via_telegram
+ * INPUTS
+ *	 notification string to send to Telegram via API
+ * RESULT
+ *   returns NULL if message was sent
+ ******/
+
+function notify_via_telegram($message, $force = false) {
+	global $config;
+	if ((!isset($config['notifications']['telegram']['enabled']) || (!$force))
+	|| (!$config['notifications']['telegram']['api'] || !$config['notifications']['telegram']['chatid'])) {
```

Comment:
There should be four spaces of indent before the `||`

Please use ["K&R, BSD KNF variant style" formatting](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style).